src/Directory.Build.targets

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <BuildDependsOn>EmbedPaketFiles;$(BuildDependsOn)</BuildDependsOn>
+  </PropertyGroup>
+  <Target Name="EmbedPaketFiles">
+    <ItemGroup>
+      <EmbeddedFiles Include="$([System.IO.Path]::GetFullPath('%(Compile.Identity)'))"
+        Condition="$([System.String]::Copy('%(Identity)').Contains('paket-files'))" />
+    </ItemGroup>
+    <CreateProperty Value="@(EmbeddedFiles)">  
+      <Output TaskParameter="Value" PropertyName="embed" />  
```

Comment:
This embeds the `paket-files` in the pdb since they are not in the repository. csharp and vb support `@(EmbeddedFiles)` with relative directories, but not fsharp. This would work with all of them.